components/common/IdentityForm.vue

```diff
@@ -288,6 +323,14 @@ const hasIdentity = computed(() => {
   )
 })
 
+let Createidentity
```

Comment:
Never let

Reply:
replaced this with v-if="!hasIdentity" within button